File: Extra Credit/Third lightbulb's the charm/Fib.java
package fib;

public class Fib {
	
	public static int fibNum(int n) {
		
		int first = 0; 
		int second = 1; 
		int next; 
		
		if(n == 0) {
			return first;
			
		}
		
		for (int i = 2; i <= n; ++i) {
			
			next = first + second; 
			first = second; 
			second = next; 
			
		}
		return second;
		
	}
	
	public static void main(String args[]) {
		
		int n = 11;
		System.out.print(fibNum(n));
		
	}
	
	

}

File: Extra Credit/Third lightbulb's the charm/fib.py
def fib_num(n):
    first = 0
    second = 1

    if n == 0:
        return first

    for i in range(2, n + 1):
        next = first + second
        first = second
        second = next

    return second


if __name__ == "__main__":

    n = 11
    print(fib_num(n))